code/_onclick/hud/screen_objects.dm

```diff
@@ -382,7 +382,7 @@
 
 		if("Toggle burst fire")
 			var/obj/item/weapon/gun/G = user.get_held_item()
-			if(istype(G)) G.toggle_burst()
+			if(istype(G)) G.use_toggle_burst()
```

Comment:
this is such an ugly file generally, but i’d still prefer the control statement be on a different line